mllib/src/main/scala/org/apache/spark/ml/feature/LabeledPoint.scala

```diff
@@ -30,8 +28,12 @@ import org.apache.spark.ml.linalg.Vector
  * @param features List of features for this data point.
  */
 @Since("2.0.0")
-@BeanInfo
 case class LabeledPoint(@Since("2.0.0") label: Double, @Since("2.0.0") features: Vector) {
+
+  def getLabel: Double = label
```

Comment:
These are added to explicitly add what the companion BeanInfo class was implicitly adding